pulsar-opentelemetry/src/main/java/org/apache/pulsar/opentelemetry/OpenTelemetryService.java

```diff
@@ -72,7 +72,9 @@ public OpenTelemetryService(String clusterName,
         sdkBuilder.addPropertiesSupplier(() -> Map.of(
                 OTEL_SDK_DISABLED_KEY, "true",
                 // Cardinality limit includes the overflow attribute set, so we need to add 1.
-                "otel.experimental.metrics.cardinality.limit", Integer.toString(MAX_CARDINALITY_LIMIT + 1)
+                "otel.experimental.metrics.cardinality.limit", Integer.toString(MAX_CARDINALITY_LIMIT + 1),
+                // Reduce number of allocations by using reusable data mode.
+                "otel.java.experimental.exporter.memory_mode", "reusable_data"
```

Comment:
Do the libraries contain constants that could be referenced?

Reply:
They don't for the config names, unfortunately: https://github.com/open-telemetry/opentelemetry-java/blob/4d8f4f31111cfac2ad8e0e467f604318ae88e3ab/exporters/common/src/main/java/io/opentelemetry/exporter/internal/ExporterBuilderUtil.java#L44.

I swapped out the value with the name of the enum, good catch!